src/sandboxpurpur/main.cpp

```diff
@@ -1,138 +1,51 @@
-#define GLFW_INCLUDE_NONE
+#include "../silly_sprites/event/events.hpp"
+#include "../silly_sprites/include/silly_sprites.hpp"
 
-#include "../silly_sprites/input.hpp"
-#include "../silly_sprites/opengl/buffer_object.hpp"
-#include "../silly_sprites/opengl/shader_program.hpp"
-#include "../silly_sprites/opengl/window.hpp"
-#include "../silly_sprites/stopwatch.hpp"
-#include "../silly_sprites/types.hpp"
-#include "magic_enum_wrapper.hpp"
-#include <glad/gl.h>
+#include <iostream>
 
-/*
-static char const* const example_vertex_shader{ R"(
-        #version 330 core
-        layout (location = 0) in vec3 aPos;
+using namespace sly;
 
-        void main()
-        {
-            gl_Position = vec4(aPos.x, aPos.y, aPos.z, 1.0);
-        }
-    )" };
-*/
-static char const* const example_vertex_shader{ "              " };
-/*
-static char const* const example_fragment_shader{ R"(
-        #version 330 core
-        out vec4 FragColor;
-        
-        void main()
-        {
-            FragColor = vec4(1.0f, 0.5f, 0.2f, 1.0f);
-        }
-    )"
+class SandboxApplication final : public sly::Application {
+public:
+    SandboxApplication() : sly::Application{ sly::ApplicationSettings{} } { }
 };
-*/
-static char const* const example_fragment_shader{ R"(
-        #version 330 core
-        out vec4 FragColor;
-        
-        void main()
-        
-            FragColor = vec4(1.0f, 0.5f, 0.2f, 1.0f);
-        }
-    )" };
 
-static inline std::vector<sly::gl::Vertex> points{
-    // clang-format off
-    sly::gl::Vertex{ glm::vec3{  0.0f,  0.5f, 0.0f } }, // top
-    sly::gl::Vertex{ glm::vec3{ -0.25f, 0.0f, 0.0f } }, // mid left
-    sly::gl::Vertex{ glm::vec3{  0.25f, 0.0f, 0.0f } }, // mit right
-    sly::gl::Vertex{ glm::vec3{ -0.5f, -0.5f, 0.0f } }, // bottom left
-    sly::gl::Vertex{ glm::vec3{  0.0f, -0.5f, 0.0f } }, // bottom mid
-    sly::gl::Vertex{ glm::vec3{  0.5f, -0.5f, 0.0f } }, // bottom right
-    // clang-format on
-};
-
-static inline std::vector<unsigned int> const indices{
-    0, 1, 2, // top
-    1, 3, 4, // bottom left
-    2, 4, 5, // bottom right
-};
-
-void move_points_down(sly::gl::BufferObject& buffer) {
-    for (auto& vertex : points) {
-        vertex.position.y -= 0.01f;
+class Object : public event::Handler<event::Sound>, public event::Handler<event::Message> {
+public:
+    Object() {
+        event::Sound::connect(this);
```

Comment:
I'm trying to understand how to use the new event system, but I don't understand why you explicitly call `connect()` for the `Sound` event, but not for the `Message` event. Does this lead to the instance receiving `Sound` events twice, but `Message` events only once?